src/games/strategy/triplea/ai/proAI/util/ProBattleUtils.java

```diff
@@ -207,17 +215,17 @@ public ProBattleResult callBattleCalculator(final PlayerID player, final Territo
     final int runCount = Math.max(16, 100 - minArmySize);
     if (isAttacker) {
       if (retreatWhenOnlyAirLeft) {
-        ai.getCalc().setRetreatWhenOnlyAirLeft(true);
+        calc.setRetreatWhenOnlyAirLeft(true);
```

Comment:
Outside of the AI code. Its essentially a property of the battle calc that's a boolean so not really an issue.